test/e2e/upgrade/upgrade.go

```diff
@@ -157,11 +167,24 @@ var _ = g.Describe("[sig-arch][Feature:ClusterUpgrade]", func() {
 			upgradeTests,
 			func() {
 				for i := 1; i < len(upgCtx.Versions); i++ {
-					framework.ExpectNoError(clusterUpgrade(f, client, dynamicClient, config, upgCtx.Versions[i]), fmt.Sprintf("during upgrade to %s", upgCtx.Versions[i].NodeImage))
+					upgradeErr = clusterUpgrade(f, client, dynamicClient, config, upgCtx.Versions[i])
+					framework.ExpectNoError(upgradeErr, fmt.Sprintf("during upgrade to %s", upgCtx.Versions[i].NodeImage))
 				}
 			},
 		)
 	})
+
+	g.It("Cluster upgrade should complete successfully", func() {
+		framework.ExpectNoError(upgradeErr)
+	})
+
+	g.It("Cluster should be upgradeable after finishing upgrade", func() {
```

Comment:
what is the intended delta from https://github.com/openshift/origin/blob/master/test/e2e/upgrade/upgrade.go#L578-L590?